### PR TITLE
v4: local ij functions to v4 cython

### PR DESCRIPTION
**Pull request by isaacbrodsky**

### CODE REVIEW

**Review by kylebarron — APPROVED** [review]

File: src/h3/api/_api_template.py [hunk 1]
```diff
@@ -855,7 +855,7 @@ def experimental_h3_to_local_ij(self, origin, h):
         origin = self._in_scalar(origin)
         h = self._in_scalar(h)
 
-        i, j = _cy.experimental_h3_to_local_ij(origin, h)
+        i, j = _cy.cell_to_local_ij(origin, h)
```
Comment (ajfriend) [inline src/h3/api/_api_template.py:858]: Same here. I'd like to make no changes to this file at this stage.

We will change all these function names (and the broader API as well), just not at this stage of the migration.

**Review by ajfriend — CHANGES_REQUESTED** [review]

**Comment by ajfriend** [comment]
Also note that we're probably going to rework the Cython API (e.g., #236), so these functions will get significant changes later on anyway. So I'd rather defer the discussion of changes around these functions to the next stage.

**Comment by ajfriend** [comment]
@isaacbrodsky, so I think I got you to entertain my admittedly idiosyncratic request to remove the Cython function name changes in

- `src/h3/_cy/cells.pxd`
- `src/h3/_cy/cells.pyx`

and the Python function name changes in

- `src/h3/_cy/__init__.py`
- `src/h3/api/_api_template.py`

**Review by ajfriend — APPROVED** [review]
lgtm! thanks!

**Pull request merged**